### PR TITLE
Fix benchmarks

### LINKED ISSUE #305: Benchmarks are failing
All of the benchmarks in this repo are failing, it seems files needed in the testdata directory were removed at some point,

### PR DESCRIPTION
**Pull request by AlekSi**
Fixes #305 (which wasn't actually fixed before).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ go:
 go_import_path: github.com/prometheus/tsdb
 
 script:
-- go test -timeout 5m ./...
+- go test -v -bench . -benchtime 1ms -timeout 5m ./...
```
Comment (brian-brazil) [inline .travis.yml]: We don't run benchmarks as part of CI

Comment (AlekSi) [reply .travis.yml]: > Just to make sure they still work, not for numbers.

**Comment by krasi-georgiev** [comment]
https://github.com/prometheus/tsdb/issues/235

eventually we should have some proper benchmarking in the CI, but in the meantime I don't think this gives us any real benefit here.

**Comment by AlekSi** [comment]
The real benefit is that with that PR code in the repository actually _runs_, and also will run in the future – Travis CI will prevent future broken PRs from being merged. Without it, the code is broken. It is not about proper benchmarking in the CI at all.

**Comment by krasi-georgiev** [comment]
don't conciser this so crucial to have a separate test, but for the sake of the good spirit don't mind if it gets merged if we get one more :+1:

**Comment by brian-brazil** [comment]
Benchmarks can take a really long time to run, we shouldn't have them slowing down CI.

**Comment by AlekSi** [comment]
That's why I added `-benchtime 1ms`. As you can see ([[1]](https://travis-ci.org/prometheus/tsdb/jobs/379086209), [[2]](https://travis-ci.org/prometheus/tsdb/jobs/379164140)) this increased CI time from 1:22 to 2:00. That sounds acceptable to me.

**Comment by brian-brazil** [comment]
As that result indicates, it's not. All of our travis configs should be consistent, and we have benchmarks elsewhere that take a lot longer than 38 seconds.

**Comment by krasi-georgiev** [comment]
oh if it increases the the tests by more than 1-2sec I am also against it.

**Comment by krasi-georgiev** [comment]
I am not sure about this one , on one side @AlekSi is right that we want to make sure the bench tests work, but on another we want to keep the test times really minimal and running this locally increases the running time by 2min and not sure if this also might cause hitting some Travis limits at some point.

@fabxc , @gouthamve what do you think on this one?

**Comment by AlekSi** [comment]
Extracted the less controversial bit into #345.

**Comment by krasi-georgiev** [comment]
@AlekSi would you mind if we close this and can revisit at some point when we have a better plans for benchmarks which is tracked  in issue https://github.com/prometheus/tsdb/issues/235

**Pull request closed without merging**